docs/reference/docs/reindex.asciidoc

```diff
@@ -436,7 +436,10 @@ POST _reindex
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=refresh]
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeout]
+`timeout`::
+(Optional, <<time-units, time units>>) Time period to wait for
```

Comment:
This `timeout` is unlike other timeouts. It is passed to the underlying bulk requests. See my comment on bulk.